src/inputs/Table/components/TableToolbar/TableToolbar.spec.js

```diff
@@ -0,0 +1,113 @@
+
+import React from 'react';
+import { TableToolbar } from './TableToolbar';
+import { renderInMuiThemeProvider } from '../../../../../tests/utils';
+import { StackContainerTesting } from '../../../../../tests/MuiComponentsTesting';
+
+const tableToolbarInputContainer = new StackContainerTesting({ dataCy: 'table-toolbar' });
+const FullscreenButtonContainer = new StackContainerTesting({ dataCy: 'grid-fullscreen-button' });
+const ImportButtonContainer = new StackContainerTesting({ dataCy: 'import-file-button' });
+const ExportButtonContainer = new StackContainerTesting({ dataCy: 'export-button' });
+const AddRowButtonContainer = new StackContainerTesting({ dataCy: 'add-row-button' });
+const DeleteRowButtonContainer = new StackContainerTesting({ dataCy: 'delete-rows-button' });
+
+const defaultProps = {
+  isFullscreen: false,
+  toggleFullscreen: () => {},
+  isReady: false,
+  editMode: true,
+};
+const setUp = (props) => {
+  renderInMuiThemeProvider(<TableToolbar {...props} />);
+};
+
+describe('Check if TableToolbar buttons is displayed or not', () => {
+  test('Check if tableToolbar is displayed with default data', () => {
+    setUp(defaultProps);
+    expect(tableToolbarInputContainer.Stack).toBeInTheDocument();
+  });
+  test('Check if open fullscreen button is displayed with default values', () => {
+    setUp(defaultProps);
+    expect(FullscreenButtonContainer.Stack).toBeInTheDocument();
+  });
+
+  describe('Check if ImportButton is displayed or not', () => {
+    test('Check if ImportButton is not displayed with default data', () => {
+      setUp(defaultProps);
+      expect(ImportButtonContainer.Stack).not.toBeInTheDocument();
+    });
+    test('Check if ImportButton is not displayed with just editMode=false', () => {
+      setUp({ ...defaultProps, editMode: false });
+      expect(ImportButtonContainer.Stack).not.toBeInTheDocument();
+    });
+    test('Check if ImportButton is disabled with onImport function and default value', () => {
+      setUp({ ...defaultProps, onImport: () => {} });
+      expect(ImportButtonContainer.Stack).toBeInTheDocument();
+      expect(ImportButtonContainer.Stack).not.toHaveAttribute('aria-disabled');
+    });
+    test('Check if ImportButton is disabled with editMode=false and onImport function', () => {
+      setUp({ ...defaultProps, editMode: false, onImport: () => {} });
+      expect(ImportButtonContainer.Stack).toHaveAttribute('aria-disabled');
+    });
+  });
+
+  describe('Check if ExportButton is displayed or not', () => {
+    test('Check if ExportButton is not displayed with default data', () => {
+      setUp(defaultProps);
+      expect(ExportButtonContainer.Stack).not.toBeInTheDocument();
+    });
+    test('Check if ExportButton is displayed with onExport function and default value', () => {
+      setUp({ ...defaultProps, onExport: () => {} });
+      expect(ExportButtonContainer.Stack).toBeInTheDocument();
+      expect(ExportButtonContainer.Stack);
```

Comment:
This `expect` seems not to do anything, is it here on purpose?

Reply:
Fixed